Add post for deferring a case (#531)
NHSDigital/prevention-services-design-history: app/manage-breast-screening/2026/06/deferring-a-case/index.md

---
title: Deferring a case from reading

> description: Sdfesf

Reviewer: 👀😂

> date: 2026-06-15

Reviewer 2: ```suggestion
date: 2026-06-16
```

author: Ed Horsford
opengraphImage:
  src: /manage-breast-screening/2026/06/deferring-a-case/deferral-reason.png

>   alt: A page shown to collect the reason for deferral. It has a textarea where the image reader can write a decription.

Reviewer 2: ```suggestion
  alt: A page shown to collect the reason for deferral. It has a text area where the image reader can write a description.
```

Reviewer 2: Not sure if text area should be all one word as it refers to a component?

tags:
  - beta
  - prototype
  - image reading
---

We’ve been looking into how we support image readers in flagging a case that has an issue and cannot be read, and how an admin interface could work for showing these cases.

## Things that might go wrong on a case

> There are many checks that happen at each stage of screening to minimise the chance of something going wrong. With our new service, some of those will hopefully be unnecessary or can be automated. However, we need to allow some flexibility to cope if things go wrong, for example:

Reviewer 2: ```suggestion
There are many checks that happen at each stage of screening to minimise the chance of images not being readable by the time they are looked at by radiologists. With our new service, some of those will hopefully be unnecessary or can be automated. However, we need to allow some flexibility to cope if things go wrong, for example:
```

Reviewer 2: Just specifies what the 'something going wrong' relates to in this instance

- The images don’t appear to match prior images, implying they’re from a different participant
- The images are mislabeled (this might not always prevent reading, but might indicate another issue)
- Images are missing without explanation
- Some other data issue that implies something has gone wrong.

## The current process

> Breast screening offices read clinic by clinic, typically with paper bundles for each clinic, but sometimes using NBSS on its own (‘paper light’). Administrative staff will check over the clinics and their bundles ahead of time to prepare them. If they come across any anomalies / issues, they can pull the case from the bundle to be reviewed so it doesn’t go to image reading. This is sometimes called an ‘exception list’. If a case with issues made it to image reading, the reader could also refuse to read it and place the case aside to be reviewed.

Reviewer: ```suggestion
Breast screening offices read clinic by clinic, typically with paper bundles for each clinic, but sometimes using NBSS on its own (‘paper light’). Administrative staff will check over the clinics and their bundles ahead of time to prepare them. If they come across any anomalies or issues, they can pull the case from the bundle to be reviewed so it doesn’t go to image reading. This is sometimes called an ‘exception list’. If a case with issues made it to image reading, the reader could also refuse to read it and place the case aside to be reviewed.
```

> Breast screening offices read clinic by clinic, typically with paper bundles for each clinic, but sometimes using NBSS on its own (‘paper light’). Administrative staff will check over the clinics and their bundles ahead of time to prepare them. If they come across any anomalies / issues, they can pull the case from the bundle to be reviewed so it doesn’t go to image reading. This is sometimes called an ‘exception list’. If a case with issues made it to image reading, the reader could also refuse to read it and place the case aside to be reviewed.

Reviewer: This is quite a long paragraph. Maybe split it up?

## Allowing for flexibility

> The great thing about paper is its flexibility. If you need a new checkbox field, you can just draw one. If you need to set a case aside, you move the paper. With the move towards [reading by session](https://design-history.prevention-services.nhs.uk/manage-breast-screening/2026/02/reading-in-batches/) (previously called batch) and automatically putting oldest cases first for reading, what’s our equivalent?

Reviewer 2: ```suggestion
The great thing about paper is its flexibility. If you need a new checkbox field, you can just draw one. If you need to set a case aside, you move the paper. With the shift towards [reading by session](https://design-history.prevention-services.nhs.uk/manage-breast-screening/2026/02/reading-in-batches/) (previously called batch) and automatically putting oldest cases first for reading, what’s our equivalent?
```

Reviewer 2: avoid double use of 'move'

> Our reading interface currently only supports an image reader giving an opinion or requesting priors - but they have no way of flagging the case / putting it aside. They can skip to the next case, but because we always put oldest case first, the case with the issue will keep being offered. We need an equivalent way that image readers can mark the case to be reviewed.

Reviewer 2: ```suggestion
Our reading interface currently only supports an image reader giving an opinion or requesting priors - but they have no way of flagging the case or putting it aside. They can skip to the next case, but because we always put oldest case first, the case with the issue will keep being offered. We need a way for image readers to mark the case to be reviewed.
```

### Deferring from the opinion page

![Image reading opinion page with link at the bottom below medical summary that reads ‘defer this case’](opinion-page-with-issue.png)

We’ve added ‘Defer this case’ as a link at the bottom of the page. The link doesn’t need to be prominent or always visible, just available for the rare time something has gone wrong. This unobtrusive placement reflects the expectation that users should be giving an opinion for the vast majority of cases (even when it's a difficult case to read) and shouldn't be deferring unless there is an issue to be resolved.

In this example there are two issues: the MLO images appear to be swapped (what’s labelled as an RMLO is an LMLO and visa versa), and there is a missing LCC and no image note about it being partial mammography.

### Giving a deferral reason

![A page shown to collect the reason for deferral. It has a textarea where the image reader can write a decription.](deferral-reason.png)

> The image reader will be asked for a description of what is wrong. This will be available to administrative staff when they are reviewing this case.

Reviewer 2: ```suggestion
When deferring, image readers must provide a description of what is wrong. This will be available to administrative staff when they are reviewing this case.
```

Once requested, the UI moves on to the next case to read. Behind the scenes we block the case from being read and put it into a workflow to be reviewed. From the image reader’s point of view doing a session of 25 cases, this will count as completed - the same as when they request priors.

### Session overview

![A portion of the session overview showing a list of participants whose cases have been read, one that is deferred, and the next case to be read](session-overview.png)

The case will show as deferred on the session overview, and image readers can revisit that case and change their answer.

### Image reading homepage

> ![The image reading dashboard an inset text block if there are deferred cases.](image-reading-homepage .png)

Reviewer: This image isn’t working.

> ![The image reading dashboard an inset text block if there are deferred cases.](image-reading-homepage .png)

Author: ```suggestion
![The image reading dashboard an inset text block if there are deferred cases.](image-reading-homepage.png)
```

When there are deferred cases, these get highlighted on the image reading dashboard. We'll probably find a better home for these in the future as this dashboard is primarily for image readers rather than administrative staff.

### Deferral list

![A page showing any deferred cases. It includes details of the case and the deferral reason provided by the image reader. There is a link to ‘unflag case’ that can be used when the issue is resolved.](deferral-list-unresolved.png)

We’ve added a new page to show the cases that are currently deferred. We’ll likely need to do expand this page in the future, but for now it is a list of the current cases and includes the reason each is deferred. When the link ‘unflag case’ is used, the case will be removed from the deferral list and returned to the reading queue.

> We don’t currently have the ability to add any further notes - either whilst the case is being investigated or once it’s resolved - but might anticipate that’s something that may be needed.

Reviewer: This is a slightly awkwardly worded phrase: “but might anticipate that’s something that may be needed”

Consider making it more of an active phrase like “we anticipate this will be needed in future”?

> We don’t currently have the ability to add any further notes - either whilst the case is being investigated or once it’s resolved - but might anticipate that’s something that may be needed.

Author: ```suggestion
We don’t currently have the ability to add any further notes - either whilst the case is being investigated or once it’s resolved - but anticipate this may will be needed in future.
```
